auth/login/auth-login.component.ts

```diff
@@ -128,6 +130,16 @@ export class AuthLoginComponent {
     }
 
 
+    openPrivacyLink() {
+        if(this.privacyLinkDefined) {
+            window.open(environment.zeta.getPrivacyLink(this.i18n.language), '_blank').focus();
+        }
+        else{
+            window.alert("PrivacyLink is not Defined");
```

Comment:
better use a console.log here. For a severe problem it is okay to interrupt the user's flow with a window.alert, but I think, this is not the case here. What do you think?

Reply:
the comment is realize with the last commit  windows. alert wird jetzt console.log("PrivacyLink is not Defined"); verwendet